ui/components/app/transaction-decoding/components/decoding/address/address.component.js

```diff
@@ -19,16 +22,29 @@ const Address = ({
   const [showNicknamePopovers, setShowNicknamePopovers] = useState(false);
 
   const addressBook = useSelector(getAddressBook);
-  const addressBookEntryObject = addressBook.find(
-    (entry) => entry.address === checksummedRecipientAddress,
+  const getNickName = (address) => {
+    const addressBookEntryObject = addressBook.find((entry) => {
+      return address.toLowerCase() === entry.address.toLowerCase();
+    });
+    return (addressBookEntryObject && addressBookEntryObject?.name) || '';
```

Comment:
I think this can just be `return addressBookEntryObject?.name || '';`

Reply:
Actually, I don't think a function is needed here. I think the previous code with the addition of the `toLowerCase` calls should work. That should make this simpler

Reply:
@danjm Done [here](https://github.com/MetaMask/metamask-extension/pull/16018/commits/3efda348801dc6c2df0eca58c405556895c51e7f).